paddle/fluid/operators/ngraph/ops/accuracy_op.h

```diff
@@ -0,0 +1,65 @@
+/*Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#pragma once
+
+#include <string>
+#include <vector>
+#include "ngraph/ngraph.hpp"
+#include "paddle/fluid/platform/ngraph_helper.h"
+
+namespace paddle {
+namespace operators {
+namespace ngraphs {
+
+void BuildAccuracyNode(
```

Comment:
I saw this function seems common, could you extract it to helper?

Reply:
This function is to map paddle accuracy operator to ngraph accuracy op, not really a common one.  Did I understand anything wrong here?

Reply:
It seems each new op would have function BuildXXXNode, I thought we could extract any functions to reuse code.